### ISSUE TITLE
Is it normal if the composer takes 4GB of memory?

### ISSUE THREAD

**Issue by dawwa** [microsoft/BotFramework-Composer#8449]
## Describe the bug

Is it normal if the composer takes 4GB of memory?
The UI stuck after starting our bots (we have one root bot plus one skill bot), I could barely move my cursor in the web chat; and when I check the resource consumption, it seems taking lots of resources.

![image](https://user-images.githubusercontent.com/44714043/126591112-ee6b0386-1e7e-40fc-bf04-c7111e61bfa3.png)

## Version
2.0

## Browser

- [ ] Electron distribution
- [X] Chrome
- [ ] Safari
- [ ] Firefox
- [X] Edge

## OS

- [ ] macOS
- [X] Windows
- [ ] Ubuntu

**Comment by srinaath** [comment]
@dawwa Are you using the recognizer Orchestrator while adding the skill bot to the solution? If so, the localized model file of Orchestrator might be the reason for the bloated memory.

Cc @daveta @taicchoumsft

**Comment by dawwa** [comment]
@srinaath, yes, I am using Orchestrator, but this does not always happen, occasionally, and I have to kill the process and restart the Composer and it may just behave normal without stuck
